debug.xml

```diff
@@ -65,10 +65,15 @@
     <!-- DATABASE CONFIG -->
 
     <!--<entry key='database.driverFile'>hsqldb.jar</entry>-->
-    <entry key='database.driver'>org.h2.Driver</entry>
+    <!--<entry key='database.driver'>org.h2.Driver</entry>
     <entry key='database.url'>jdbc:h2:./target/database</entry>
     <entry key='database.user'>sa</entry>
-    <entry key='database.password'></entry>
+    <entry key='database.password'></entry>-->
+
+    <entry key='database.driver'>com.mysql.jdbc.Driver</entry>
```

Comment:
how do I exclude a change in a file?

On Wed, Jul 13, 2016 at 1:46 AM, Anton Tananaev <notifications@github.com>
wrote:

> In debug.xml
> <https://github.com/tananaev/traccar/pull/2104#discussion_r70536399>:
>
> >      <entry key='database.url'>jdbc:h2:./target/database</entry>
> >      <entry key='database.user'>sa</entry>
> > -    <entry key='database.password'></entry>
> > +    <entry key='database.password'></entry>-->
> > +
> > +    <entry key='database.driver'>com.mysql.jdbc.Driver</entry>
>
> Please exclude this change.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tananaev/traccar/pull/2104/files/d045c1e2174146ddc8ad8b2ef4442863f3d84a11#r70536399>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AH9fF1YfAWofjwO44_zCIi_9dqsseieJks5qVBkrgaJpZM4JKXy->
> .
>